.pylintrc

```diff
@@ -70,7 +70,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
-disable=consider-using-ternary,locally-disabled,locally-enabled,logging-not-lazy,abstract-method,arguments-differ,broad-except,duplicate-code,fixme,len-as-condition,missing-docstring,no-else-return,no-member,no-self-use,not-context-manager,redefined-variable-type,too-many-arguments,too-many-boolean-expressions,too-many-branches,too-many-instance-attributes,too-many-lines,too-many-locals,too-many-public-methods,too-many-nested-blocks,too-many-statements
+disable=consider-using-ternary,locally-disabled,locally-enabled,logging-not-lazy,abstract-method,arguments-differ,broad-except,duplicate-code,fixme,inconsistent-return-statements,invalid-name,len-as-condition,missing-docstring,no-else-return,no-member,no-self-use,not-context-manager,redefined-variable-type,too-many-arguments,too-many-boolean-expressions,too-many-branches,too-many-instance-attributes,too-many-lines,too-many-locals,too-many-public-methods,too-many-nested-blocks,too-many-statements,wrong-import-order
```

Comment:
@apb7, disabling `wrong-import-order` was intentional here?

Reply:
@apb7 If required, I can undo these changes.

Reply:
I think I have explained this in the PR description. Moreover, `wrong-import-order` has a couple of bugs filed against it: https://github.com/PyCQA/pylint/issues/2175. Thanks!

Reply:
I missed the Explanation section of this PR, thanks for pointing. Still, I don't understand why we have enabled/disabled this in few files?
![image](https://user-images.githubusercontent.com/16653571/54865809-4aad9f80-4d91-11e9-9256-0b70ee77ba9e.png)